andela/wg-millennium-falcon
# #157731564 List active and Inactive users in gym overview

#### What does this PR do?
List active versus inactive users on the users list and in the gym overview.

#### Description of Task to be completed?
When an Administrator views the users list or drills down to a gym to view the users list,
they can switch between active and inactive users.
Also, the list is now paginated.
![image](https://user-images.githubusercontent.com/7848682/40741565-89e605c4-6454-11e8-813a-b1d71ea5b6c3.png)
The above screenshot shows a list of Inactive users paginated at 10 users per view.

![image](https://user-images.githubusercontent.com/7848682/40741634-b1f6b89c-6454-11e8-9739-ba66794eddfa.png)
The above screenshot shows a list of Active users paginated by default at 7 users per view.

#### How should this be manually tested?
Log into Wger as an Administrator and checkout the users list or the gym overview.
You should be able to choose between Active and Inactive users.

#### Any background context you want to provide?
Presently, In the gym overview, Wger displays all the users 
but can does not show if a user is active or not unless you open the user's detail view.

#### What are the relevant pivotal tracker stories?

[#157731564](https://www.pivotaltracker.com/story/show/157731564)
